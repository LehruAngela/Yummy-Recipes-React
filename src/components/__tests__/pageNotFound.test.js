import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import PageNotFound from '../pageNotFound';

describe('PageNotFound', () => {
    const wrapper = shallow(<PageNotFound/>);
    const preventDefault = jest.fn();

    it('renders properly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
        }); 
    

})