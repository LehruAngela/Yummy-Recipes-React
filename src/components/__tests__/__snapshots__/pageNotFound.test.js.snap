// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders properly 1`] = `
<h1>
   Page Not Found 
</h1>
`;
